### ISSUE TITLE
transfer_pokemon: Show candies with pokemon name without evolve

### ISSUE THREAD

**Issue by LazGit** [PokemonGoF/PokemonGo-Bot#5459]
**Expected Behavior**
Keeping best 1 Pidgeotto, based on iv
\*Pidgeotto Released\* You now have 4 Pidgey candies

**Actual Behavior**
Keeping best 1 Pidgeotto, based on iv
\*Pidgeotto Released\* You now have 4 Pidgeotto candies

**Steps to Reproduce**
Update to the latest dev commit

**Comment by davidspeijer** [comment]
I have seen the change, however isn't the issue that pidgeotto candy is in the log instead of pidgey candy?

**Comment by javajohnHub** [comment]
I see. I misunderstood.

**Comment by Anakin5** [comment]
There is no such thing as "Pidgeotto candies". Candies for Pokemon of the same family bear the same name and that is the name of the non evolved Pokemon.
Pidgey, Pidgeotto and Pidgeot all use "Pidgey candies"

**Comment by Gobberwart** [comment]
I think that's the point. Log is showing pidgeotto where it should be showing pidgey.

**Comment by Anakin5** [comment]
oh ok, I got it reversed

**Comment by Gobberwart** [comment]
Fixed:

```
[2016-09-22 13:13:41] [TransferPokemon] [INFO] Keeping best 2 Raticate, based on ivcp and cp
[2016-09-22 13:13:41] [TransferPokemon] [INFO] Kept Raticate (CP: 1134, IV: 0.56, IVCP: 0.556857221826)
[2016-09-22 13:13:41] [TransferPokemon] [INFO] Kept Raticate (CP: 1120, IV: 0.73, IVCP: 0.66114254634)
[2016-09-22 13:13:41] [TransferPokemon] [INFO] Released Raticate (CP: 1001, IV: 0.67, IVCP: 0.65) You now have 403 Rattata candies
```
